autoload/ale.vim

```diff
@@ -109,6 +109,7 @@ function! s:Lint(buffer, should_lint_file, timer_id) abort
     \   || (a:should_lint_file && filereadable(expand('#' . a:buffer . ':p')))
 
     call ale#engine#RunLinters(a:buffer, l:linters, l:lint_file)
+    call ale#code_lens#Request(a:buffer)
```

Comment:
Check if the lens feature is enabled before calling functions from `code_lens.vim`, and then the code won't be loaded for users who don't turn the feature on.